### ISSUE TITLE
More documentation and spelling

### ISSUE THREAD

**Issue by WaterGazes** [8954sood/youtubedown#1]
Hello, me and a few others would probably appreciate if there could be some more documentation on this lib. There are also a few spelling errors in the documentation that I would like to inform you of.

**Comment by 8954sood** [comment]
Thank you for attaching the first commitment of this module. We will accept the request and write about the module's return. However, there is a problem with this module, so I would appreciate it if you could use it for reference only.

**Comment by WaterGazes** [comment]
Hopefully you can fix it soon because this is the only async lib I could find! Also if you're looking for suggestions I think a good place to start would be to make an option for video quality and video format.

**Comment by 8954sood** [comment]
``
loop = asyncio.get_event_loop()
await loop.run_in_executor
``
Please note that the above code allows synchronization code to be executed asynchronously at all the time.

**Comment by 8954sood** [comment]
Thanks to this, I became motivated to develop.
We will add a way to set up the video format and download options you suggested.
To be honest, I made it thinking that no one would use this module. As a result, I just passed it even though I knew the problem of the module. We're working hard to improve this problem.
We will add return examples, video formats, and options as soon as possible and fix the problem.
From a beginner developer.
